spec/database.yml

```diff
@@ -27,6 +27,7 @@ postgresql:
 mysql:
   adapter: seamless_database_pool
   database: seamless_database_pool_test
+  prepared_statements: false
```

Comment:
This will default to true, so I think this needs to be explicitly set for MySQL.